pl_examples/basic_examples/gpu_template.py

```diff
@@ -68,6 +72,24 @@ def main(hparams):
         action='store_true',
         help='if true uses 16 bit precision'
     )
+    parent_parser.add_argument(
+        '--evaluate_val',
+        dest='evaluate_val',
+        action='store_true',
+        help='evaluate on validation set'
```

Comment:
what is the relationship between test and val, can I run both or just one of them?

Reply:
both can be run. will change it.

Reply:
then I would rather have a parameter `validation` which may have a list of types like `test`, `valid`, ...